### DISCUSSION TITLE
Having issue starting up docker env

### DISCUSSION THREAD

**Discussion by kevkevinpal** [stackernews/stacker.news#918] [local dev help]
I tried running `docker-compose up --build` but it seems that it starts to start up but then the `app` container crashes

I'm on macos not sure if that is causing the issue

Sample timeout for docker-compose
`ERROR: for worker  Container "aabc46973284" is unhealthy.` (aabc46973284) is the app container

**Comment by huumn** [comment]
I introduced a bug in migrations. I'll push the fix in a second.

**Comment by huumn** [comment]
Just push to master. Please retry with that!

**Comment by kevkevinpal** [comment]
still seems to be crashing when I log the container these are the last logs I see before it crashes but Idk how helpful they are

```
...
Run `npm audit` for details.
npm verb exit 0
npm info ok
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "stackernews", schema "public" at "db:5432"

Already in sync, no schema change or pending migration was found.

✔ Generated Prisma Client (5.1.1 | library) to ./node_modules/@prisma/client in
5.28s


┌─────────────────────────────────────────────────────────┐
│  Update available 5.1.1 -> 5.2.0                        │
│  Run the following to update                            │
│    npm i --save-dev prisma@latest                       │
│    npm i @prisma/client@latest                          │
└─────────────────────────────────────────────────────────┘

> stackernews@0.1.0 dev
> NODE_OPTIONS='--trace-warnings' next dev

- warn You have enabled experimental feature (scrollRestoration) in next.config.js.
- warn Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.

- ready started server on [::]:3000, url: http://localhost:3000
- info Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
Created precache manifest at sw/precache-manifest.json. Cache will include 103 URLs with a size of 11.49 MB.
Created precache manifest at sw/precache-manifest.json. Cache will include 103 URLs with a size of 11.49 MB.
- info Using external babel configuration from /app/.babelrc
- event compiled client and server successfully in 5.8s (18 modules)

```

**Comment by huumn** [comment]
After you pulled the latest master, did it prompt you to reset the db? It should have and these logs don't show that.

**Comment by kevkevinpal** [comment]
> After you pulled the latest master, did it prompt you to reset the db? It should have and these logs don't show that.

No I didn't see that. What should I do to reset the db?

**Comment by huumn** [comment]
You'll want to fully restart the environment or just recreate it from scratch

**Comment by kevkevinpal** [comment]
yea I do `docker-compose down` but dont see any prompt and the same issue of app crashing

**Comment by huumn** [comment]
I can try to recreate it myself tomorrow but if you're in the mood it might be worth doing `docker-compose down --rmi all -v`

**Comment by kevkevinpal** [comment]
> I can try to recreate it myself tomorrow but if you're in the mood it might be worth doing `docker-compose down --rmi all -v`

just tried that still no luck :/

**Comment by huumn** [comment]
I'll give it a shot today once I get through my daily minutia and find my groove.

**Comment by huumn** [comment]
I didn't make it around to this today as planned. Hopefully tomorrow.

**Comment by huumn** [comment]
It launched for me fine so I'm going to need more info to repro. 

What OS are you using and is there any other you might think is relevant?

**Comment by kevkevinpal** [comment]
I'm on a macbook 2019 so not an m1 mac version 13.4.1 (c) (22F770820d)

I wouldn't stress too much on this I can take some time to figure out why I'm having this issue
